dev/linters/yamllint.sh

```diff
@@ -0,0 +1,12 @@
+#!/usr/bin/env bash
+
+set -o errexit
+
+# shellcheck disable=SC2046
+# We want word splitting with find.
+bazel run //dev/linters:yamllint -- \
```

Comment:
I'm having issues running this; I get:

```
$ bazel run //dev/linters:yamllint
Starting local Bazel server and connecting to it...
DEBUG: Rule 'rules_python' indicated that a canonical reproducible form can be obtained by modifying arguments shallow_since = "1573842889 -0500"
INFO: Analyzed target //dev/linters:yamllint (18 packages loaded, 210 targets configured).
INFO: Found 1 target...
INFO: Deleting stale sandbox base …/1902c1ccc22df327a938dca0d6a953d0/sandbox
Target //dev/linters:yamllint up-to-date:
  bazel-bin/dev/linters/yamllint
INFO: Elapsed time: 14.173s, Critical Path: 0.06s
INFO: 0 processes.
INFO: Build completed successfully, 4 total actions
INFO: Build completed successfully, 4 total actions
Traceback (most recent call last):
  File "…/1902c1ccc22df327a938dca0d6a953d0/execroot/kubecf/bazel-out/k8-fastbuild/bin/dev/linters/yamllint.runfiles/kubecf/dev/linters/run_yamllint.py", line 2, in <module>
    from yamllint import cli
  File "…/1902c1ccc22df327a938dca0d6a953d0/execroot/kubecf/bazel-out/k8-fastbuild/bin/dev/linters/yamllint.runfiles/yamllint_pypi__yamllint_1_19_0/yamllint/cli.py", line 25, in <module>
    from yamllint import linter
  File "…/1902c1ccc22df327a938dca0d6a953d0/execroot/kubecf/bazel-out/k8-fastbuild/bin/dev/linters/yamllint.runfiles/yamllint_pypi__yamllint_1_19_0/yamllint/linter.py", line 19, in <module>
    import yaml
  File "…/1902c1ccc22df327a938dca0d6a953d0/execroot/kubecf/bazel-out/k8-fastbuild/bin/dev/linters/yamllint.runfiles/yamllint_pypi__PyYAML_5_1_2/yaml/__init__.py", line 374
    class YAMLObject(metaclass=YAMLObjectMetaclass):
                              ^
SyntaxError: invalid syntax
```

This appears to be because:

```
$ cat bazel-bin/dev/linters/yamllint
#!/usr/bin/env python

…
```

This appears to be https://github.com/bazelbuild/bazel/issues/4815. We seem to need to change how bazel is run to fix this:

```suggestion
bazel run --incompatible_use_python_toolchains //dev/linters:yamllint -- \
```

Looking around, maybe my version of bazel (0.26) is too old?  Ah, yes, installing bazel 0.29 (the newest version in `obs://devel:tools:building`) fixed things.  All good, then.